hypervisor/qemu/qemu.go

```diff
@@ -385,6 +406,6 @@ func (d QEMUHypervisorDriver) RebootInstance() error {
 	})
 }
 
-func (d QEMUHypervisorDriver) InstanceConsole() hypervisor.Console {
-	return nil
-}
+// func (d QEMUHypervisorDriver) InstanceConsole() hypervisor.Console {
```

Comment:
Oh this method should be removed, it's doesn't get the pointer receiver.

The correct method is implemented here:
https://github.com/axsh/openvdc/blob/19a4b597f27b9db7f983dbcda81426f9ac25242c/hypervisor/qemu/qemu_console.go#L196